docs/how-to/add-the-nginx-route-relation.md

```diff
@@ -0,0 +1,146 @@
+# How to add the Nginx-route relation.
+
+The nginx-route relation is preferred over the ingress relation if you want to use nginx-specific features, such as owasp-modsecurity-crs. If you need
+something more generic then please follow the [ingress relation](https://charmhub.io/nginx-ingress-integrator/docs/add-the-ingress-relation) tutorial instead.
+
+## Requirements
+
+You will need:
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed. We'll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
+* [Charmcraft](https://juju.is/docs/sdk/install-charmcraft) installed.
+* A code editor of your choice.
+
+## Create a charm
+
+First of all, let's create a very basic charm to add our relation to.
+```
+mkdir my-charm
+cd my-charm
+charmcraft init
+```
+You'll then see the following output:
+```
+Charmed operator package file and directory tree initialised.
+
+Now edit the following package files to provide fundamental charm metadata and other information:
+
+metadata.yaml
+config.yaml
+src/charm.py
+README.md
+```
+Let's start by updating `metadata.yaml`. We want to update the `display-name` to "My Charm", and the `summary` and `description` to "A charm with a relation to nginx ingress integrator".
+
+We can leave the other files as is for now, but if we want to publish this charm later, we'll want to update them before doing so.
+
+## Deploy the charm
+
+Now let's deploy the charm just to confirm everything is working as expected without the relation. To do that, just run this:
+```
+charmcraft pack
+```
+This will build the charm inside an LXC for you. The output will tell you the location of the built charm. In my case, that was `my-charm_ubuntu-22.04-amd64.charm`.
```

Comment:
perhaps "LXC container"?

Reply:
Perhaps avoid referring to "my case" and instead just say, for example

Reply:
Since that's a copy paste of the now renamed `ingress` relation documentation, any changes made here should be reflected on the other documentation